docs/uninstall-an-operator.md

```diff
@@ -0,0 +1 @@
+# How to uninstall an Operator?
```

Comment:
This doc is a little short 😂

Reply:
The latest version of the doc is of a more appropriate length - resolving :)